### DISCUSSION TITLE
Is there any way to sync yazi nvim colors with cli yazi?

### DISCUSSION THREAD

**Discussion by jyyhuang** [mikavilpas/yazi.nvim#191] [Q&A]
I was wondering if there was a way to sync colors between these two since my terminal emulator has a different colorscheme than nvim.

**Comment by mikavilpas** [comment]
It's a nice idea. Currently yazi might have a way to change the entire configuration directory via an environment variable. Maybe this could be used, but it might be a bit of work.
